main.go

```diff
@@ -81,6 +82,7 @@ type Config struct {
 	NSName                 string            `default:"forwarder" desc:"Name of Network Service to Register with Registry"`
 	BridgeName             string            `default:"br-nsm" desc:"Name of the OvS bridge"`
 	TunnelIP               string            `desc:"IP or CIDR to use for tunnels" split_words:"true"`
+	VxlanPort              uint16            `default:"0" desc:"VXLAN port to use" split_words:"true"`
```

Comment:
Should we use non-zero value here?

I have a concern that if we use 0 we will overwrite the default value here:
https://github.com/networkservicemesh/sdk-ovs/blob/12d73931586bc5f080b7049bd46b40b8d3f5538d/pkg/networkservice/mechanisms/vxlan/server.go#L52-L57

Reply:
My intention was to be consistent with forwarder-vpp. You're right, the default (0) value will be overwritten by our 'secret'  default (4789) :) . It would be more intuitive if the default value is 4789 in the config as well.